packages/apps/src/contexts/activity.ts

```diff
@@ -73,6 +73,24 @@ export interface IActivityContextOptions<T extends Activity = Activity> {
   next: (context?: IActivityContext) => (void | InvokeResponse) | Promise<void | InvokeResponse>;
 }
 
+interface ISignInOptions {
+  /**
+   * The name of the auth connection to use
+   * @default `graph`
+   */
+  connectionName: string;
+  /**
+   * The text to display on the oauth card
+   * @default `Please Sign In...`
+   */
+  oauthCardText: string;
+  /**
+   * The text to display on the sign in button
+   * @default `Sign In`
+   */
```

Comment:
can we change this to a `type` instead? After the `I` interface rename I have been trying to use types over interfaces when inheritance isn't being used or when a class isn't intended to implement the interface.